save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -112,7 +140,64 @@ external interface OrganizationContestsTableProps<D : Any> : TableProps<D> {
 private fun organizationContestsMenu() = FC<OrganizationContestsMenuProps> { props ->
     val (isToUpdateTable, setIsToUpdateTable) = useState(false)
     val (isContestCreationModalOpen, setIsContestCreationModalOpen) = useState(false)
+    val (isDeleteContestModalOpen, setDeleteContestModalOpen) = useState(false)
+    val (deletingContest, setDeletingContest) = useState<ContestDto?>(null)
+    val (confirmLabel, setConfirmLabel) = useState("")
+    val (confirmMessage, setConfirmMessage) = useState("")
+    val (contests, setContests) = useState<Set<ContestDto>>(setOf())
     val refreshTable = { setIsToUpdateTable { !it } }
+    val deleteCont = useDeferredRequest {
```

Comment:
```suggestion
    val deleteContest = useDeferredRequest {
```
Why abbreviate function name?

Reply:
Renamed